src/core/tests/Test_UbuntuProClient.py

```diff
@@ -45,12 +45,15 @@ def mock_unimport_module(self, module_name):
 
 
 class MockVersionResult(MockSystemModules):
-    def __init__(self, version='27.13.5~18.04.1'):
+    def __init__(self, version='27.14.4~18.04.1'):
         self.installed_version = version
 
     def mock_version(self):
         return MockVersionResult()
 
+    def mock_version_to_version_below_minimum_version(self):
```

Comment:
nit: Could this be mock_to_below_minimum_version()?

Reply:
Wil change